draft-irtf-cfrg-hpke.md

```diff
@@ -20,6 +20,10 @@ author:
     name: Karthik Bhargavan
     org: Inria
     email: karthikeyan.bhargavan@inria.fr
+ -  ins: B. Lipp
+    name: Benjamin Lipp
+    org: INRIA Paris
```

Comment:
```suggestion
    org: Inria
```

Let's do `Inria`, because I only recently learned that this is the new preferred writing of the institute. And Karthik does not have `Paris`, so I'd drop that too for consistency.